### ISSUE TITLE
Errors after login

### ISSUE THREAD

**Issue by jrdh** [NaturalHistoryMuseum/scratchpads2#5788]
Specifically:

```
Notice: Undefined index: external_scripts in theme_remote_issue_tab_block() (line 17 of /var/aegir/platforms/scratchpads-2.9.4/sites/all/modules/custom/remote_issue_tab/remote_issue_tab.theme.inc).
Warning: Invalid argument supplied for foreach() in theme_remote_issue_tab_block() (line 17 of /var/aegir/platforms/scratchpads-2.9.4/sites/all/modules/custom/remote_issue_tab/remote_issue_tab.theme.inc).
Notice: Undefined index: scripts in theme_remote_issue_tab_block() (line 21 of /var/aegir/platforms/scratchpads-2.9.4/sites/all/modules/custom/remote_issue_tab/remote_issue_tab.theme.inc).
Warning: Invalid argument supplied for foreach() in theme_remote_issue_tab_block() (line 21 of /var/aegir/platforms/scratchpads-2.9.4/sites/all/modules/custom/remote_issue_tab/remote_issue_tab.theme.inc).
Notice: Undefined index: fetch_url in theme_remote_issue_tab_block() (line 26 of /var/aegir/platforms/scratchpads-2.9.4/sites/all/modules/custom/remote_issue_tab/remote_issue_tab.theme.inc).
Notice: Undefined index: provider_module in theme_remote_issue_tab_block() (line 27 of /var/aegir/platforms/scratchpads-2.9.4/sites/all/modules/custom/remote_issue_tab/remote_issue_tab.theme.inc).
Notice: Undefined index: html_link in theme_remote_issue_tab_create_issue() (line 49 of /var/aegir/platforms/scratchpads-2.9.4/sites/all/modules/custom/remote_issue_tab/remote_issue_tab.theme.inc).
Notice: Undefined index: header in theme_remote_issue_tab_block() (line 33 of /var/aegir/platforms/scratchpads-2.9.4/sites/all/modules/custom/remote_issue_tab/remote_issue_tab.theme.inc).
Notice: Undefined index: html_link in theme_remote_issue_tab_list_issues() (line 60 of /var/aegir/platforms/scratchpads-2.9.4/sites/all/modules/custom/remote_issue_tab/remote_issue_tab.theme.inc).
```

Screenshot: ![screenshot from 2019-01-28 14-17-47](https://user-images.githubusercontent.com/4718259/51841904-91d66180-2307-11e9-9e17-63a0a99d804f.png)

**Comment by benscott** [comment]
Duplicate of #5780

**Comment by benscott** [comment]
Fixed  - #5780
